sktm/patchwork.py

```diff
@@ -237,24 +237,25 @@ def stringify(value):
     return str(value)
 
 
-# Internal RH PatchWork adds a magic API version with each call
-# this class just magically adds/removes it
 class RpcWrapper(object):
+    """
+    Class for removal of magic API version which is added by RH Patchwork fork.
```

Comment:
Saying "Class" here is redundant, and something like "XML RPC object wrapper removing magic API version..." would be better, I think. Not required for my ACK on this.

Reply:
Fixed